components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/dao/DeviceDAO.java

```diff
@@ -60,5 +60,10 @@
 	 */
 	List<Device> getDeviceListOfUser(String username , int tenantId) throws DeviceManagementDAOException;
 
-
+    /**
+     * Get the count of devices
```

Comment:
Fixed in commit [1]
[1] https://github.com/wso2/carbon-device-mgt/commit/60960d80b4cfa1e7bf16c3fffb10f9295e2185ae